release-notes/opendistro-for-elasticsearch-security.release-notes-1.13.0.0.md

```diff
@@ -0,0 +1,36 @@
+## 2021-02-17 Version 1.13.0.0
+
+Compatible with Elasticsearch 7.10.2
+
+### Enhancements
+
+* Using SAML subject_key and roles_key in the HTTPSamlAuthenticator ([#892](https://github.com/opendistro-for-elasticsearch/security/pull/892))
+* Support for ES system index ([#946](https://github.com/opendistro-for-elasticsearch/security/pull/946))
+* Updating Autheticators to throw RuntimeException on errors ([#505](https://github.com/opendistro-for-elasticsearch/security/pull/505))
+* Add security configuration for Kibana Notebooks ([#903](https://github.com/opendistro-for-elasticsearch/security/pull/903))
+* Short circuit privilege evaluation for bulk requests without index resolution ([#926](https://github.com/opendistro-for-elasticsearch/security/pull/926))
+* Added async search response index to system index list ([#859](https://github.com/opendistro-for-elasticsearch/security/pull/859))
+* Support ES 7.10.2 ([#1005](https://github.com/opendistro-for-elasticsearch/security/pull/1005))
```

Comment:
Upgrade to patch release should be in Maintenance section.

Reply:
Changed.